Core/clim-core/table-formatting.lisp

```diff
@@ -66,38 +82,57 @@
   (:default-initargs
    :align-x :left :align-y :baseline :min-width 0 :min-height 0))
 
+(eval-when (:compile-toplevel :load-toplevel :execute)
+  (alexandria:define-constant +standard-cell-output-record-initargs+
+      '(align-x align-y min-width min-height)
+    :test #'equal))
+
 (defgeneric invoke-formatting-cell
-    (stream cont &key align-x align-y min-width min-height record-type &allow-other-keys)
-  (:method (stream continuation &key
-                                  (align-x :left)
-                                  (align-y :baseline)
-                                  (min-width 0)
-                                  (min-height 0)
-                                  (record-type 'standard-cell-output-record)
-                                  &allow-other-keys)
-    (invoke-with-new-output-record
-     stream
-     (lambda (stream record)
-       (declare (ignore record))
-       (with-temporary-margins (stream :left '(:absolute 0))
-         (letf (((stream-cursor-position stream) (values 0 0)))
-           (funcall continuation stream))))
-     record-type
-     :align-x align-x
-     :align-y align-y
-     :min-width (parse-space stream min-width :horizontal)
-     :min-height (parse-space stream min-height :vertical))))
+    (stream continuation
```

Comment:
please move generic functions invoke-* to decls.lisp

add them under 17.2 with a comment, that they are not part of the specification. initargs should be manually mentioned there.

Reply:
They are ordinary functions now.